### PR TITLE
SEO for the webpage

### PR DESCRIPTION
**Pull request by lukeIam**
- Add Metadata
- Fix feed thumbnails
- Small config fixes

for #240 

@hendrikbl

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ url: "https://new.vocaluxe.org"
 #
 # NB! Without this *nothing* will work, because it's used in every path :)
```
Comment (flokuep) [inline _config.yml]: Not sure if this comment is for url or baseurl?

Comment (hendrikbl) [reply _config.yml]: I think as long as we don't use `{site.baseurl}` as a variable somewhere we should be good removing it. Referring to [this post](https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/) it's only used if the site is not hosted in the root directory of the server.

**Comment by hendrikbl** [comment]
Looks good to me :+1:
I'm currently playing around with the styles and started to clean up the CSS and convert it to Sass. It should be way easier for future changes and readability.

File: _config.yml (hunk 1 above)
Comment (lukeIam) [reply _config.yml]: looks like we have to set it to
```
baseurl: ""
```

Otherwise Github Pages sets it to `Vocaluxe`

**Pull request merged**